.github/workflows/codeql.yml

```diff
@@ -0,0 +1,42 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
Should this include wildcards like the tinymce repo?

Reply:
Don't think we need to include wildcard like `main/**` since there is only one base branch